Gemfile

```diff
@@ -39,6 +39,10 @@ gem 'bootsnap', '>= 1.1.0', require: false
 group :development, :test do
   # Call 'byebug' anywhere in the code to stop execution and get a debugger console
   gem 'byebug', platforms: [:mri, :mingw, :x64_mingw]
+  gem 'execjs'
```

Comment:
this can be removed as we discussed

Reply:
I removed the extra gems in the Gemfile 
![ImagenPullRequest2](https://user-images.githubusercontent.com/58991676/71285133-2fcd8300-2343-11ea-8262-ea5a48fa31c7.png)